python/py-barnaba/Portfile

```diff
@@ -48,6 +48,5 @@ if {${name} ne ${subport}} {
 
     depends_test-append     port:py${python.version}-nose
```

Comment:
this test dependency is automatically added when using `python.test_framework nose`

Reply:
Fixed (and force pushed). Thanks @reneeotten